cmd-export.c

```diff
@@ -122,7 +127,7 @@ int cmd_export(int argc, char **argv)
 		}
 	}
 
-	printf("url,username,password,extra,name,grouping,fav\r\n");
+	printf("id,name,group,groupname,fullname,url,username,password,note,last_touch,last_modified_gmt,fav,attachpresent\r\n");
 
```

Comment:
I don't think we can change the order here, website import might expect a certain format.  Should import also handle all of these fields?  Meta-question: if you had json output from cmd-show, would you need these fields?